src/main/java/alexandr_samohvalov/BreakContinueMain.java

```diff
@@ -0,0 +1,32 @@
+package alexandr_samohvalov;
+
+import java.util.Scanner;
+
+public class BreakContinueMain {
+    //Create a program, which will ask user to enter a number, if number will be negative exit the loop
+    public static void negativeNumExit(String[] negativeNumExit) {
+        Scanner sc = new Scanner(System.in);
+        int sum = 0;
+        for (int i = 1; i <= 7; i++) {
```

Comment:
you have to ask for inout as many time as possible while(true) not only 7 times maximum

Reply:
was not fixed